src/Reminder.php

```diff
@@ -667,6 +652,54 @@ public static function showListForCentral(bool $personal = true, bool $display =
             $public_criteria   = array_merge_recursive($public_criteria, $additional_criteria);
         }
 
+        // TOOD: would be cleaner to have two separate method like getPersonalCriteria
+        // and getPublicCriteria
```

Comment:
You can do it by passing a `bool $personal` parameter as you did for RSS feeds.

Reply:
I don't want to do it now, I've just moved the original code without too much modification as it is not the goal of this PR nor is it something high priority (and there are a lot others problems with this code that need to be fixed).